src/web_interface/templates/database/database_search.html

```diff
@@ -51,6 +53,16 @@ <h3 class="mb-3">Search Firmware Database</h3>
             </div>
             {% endfor %}
 
+            <div class="form-group">
+                <label class="control-label" for="tags">Tags:</label>
+                <select class="selectpicker" name='tags' id="tags" multiple>
+                    {% for tag in tag_list | sort %}
+                    <option value='{{ tag }}'>{{ tag }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+
+
             <button type="submit" value=submit class="btn btn-primary" id="input_submit" onclick='showImg()'>
                 <i class="fas fa-search"></i> Search
             </button>
```

Comment:
There is mixed quote usage in the html tag attributes, e.g.:

```html
<button type="submit" value=submit class="btn btn-primary" id="input_submit" onclick='showImg()'>
```

Please use double quotes here.

Reply:
fixed